internal/platform/linguakit/word.go

```diff
@@ -8,8 +12,24 @@ package linguakit
 // Where Tag=NOUN, Lemma=perro and Properties=gender:M|lemma:perro|number:S|person:3|pos:1|token:perro|type:C
 //
 type Word struct {
-	Tag        string            `json:"tag"`
-	Lemma      string            `json:"lemma"`
-	Type       string            `json:"type"`
-	Properties map[string]string `json:"properties"`
+	Tag          string            `json:"tag"`
+	Lemma        string            `json:"lemma"`
+	Type         string            `json:"type"`
+	Properties   map[string]string `json:"properties"`
+	Transitive   bool              `json:"transitive"`
+	Intransitive bool              `json:"intransitive"`
```

Comment:
IMHO this properties should not be part of the Word, because it only applies for verbs. 

What do you think? Maybe accessing directly to properties.